### PR TITLE
Server: update to reflect the latest subgraph changes

### PR DESCRIPTION
**Pull request by bpierre**

### CODE REVIEW

File: packages/govern-server/src/api-graphql/schema.ts [hunk 1]
```diff
@@ -5,21 +5,20 @@ export default gql`
...
     id: ID!
     name: String!
     executor: Dao!
     queue: OptimisticQueue!
-    metadata: String
   }
 
   type Dao {
     id: ID!
     address: String!
```
Comment (kanaagawa) [inline packages/govern-server/src/api-graphql/schema.ts:20]:
```suggestion
    address: String!
    metadata: String

File: packages/govern-server/src/api-graphql/schema.ts [hunk 2]
```diff
@@ -5,21 +5,20 @@ export default gql`
...
-    games: [OptimisticGame!]
+    registryEntries: [RegistryEntry!]
     executions: [Execution!]
```
Comment (kanaagawa) [inline packages/govern-server/src/api-graphql/schema.ts]: We removed this entity on the latests version. Instead we have containers now.

File: packages/govern-server/src/api-graphql/schema.ts [hunk 3]
```diff
@@ -5,21 +5,20 @@ export default gql`
...
     roles: [Role!]
     queues: [OptimisticQueue!]
```
Comment (kanaagawa) [inline packages/govern-server/src/api-graphql/schema.ts]: We also removed this one.

File: packages/govern-server/src/api-graphql/schema.ts [hunk 4]
```diff
@@ -45,7 +44,7 @@ export default gql`
     id: ID!
```
Comment (kanaagawa) [inline packages/govern-server/src/api-graphql/schema.ts:28]: To keep consistent with the naming we may want to rename everywhere `OptimisticQueue` for `GovernQueue`

File: packages/govern-server/src/api-graphql/schema.ts [hunk 5]
```diff
@@ -45,7 +44,7 @@ export default gql`
...
     address: String!
     config: Config
-    games: [OptimisticGame!]
+    registryEntries: [RegistryEntry!]
     queue: [Item!]
```
Comment (kanaagawa) [inline packages/govern-server/src/api-graphql/schema.ts]: All of this keep track of `Containers`.

**Review by 0xGabi — COMMENTED** [review]
Left a few comments on the schema, I think we still need to update in a few places the naming to be align with the new changes.

**Comment by 0xGabi** [comment]
I tested locally and is working fine. We still need to add a resolver to fetch queues by daos. I'm going to merge this one to check the CI/CD is working and deploy a beta version of subgraph + server to our infra

**Pull request merged**